pkg/antctl/command_definition_test.go

```diff
@@ -439,3 +441,422 @@ func TestCommandDefinitionGenerateExample(t *testing.T) {
+		},
+		{
+			name: "Command for agent defines resource endpoint",
+			cd: &commandDefinition{
+				use: "podmuticasts",
+				agentEndpoint: &endpoint{
+					resourceEndpoint: &resourceEndpoint{
+						namespaced: true,
+					},
+				},
+				controllerEndpoint:     &endpoint{},
+				flowAggregatorEndpoint: &endpoint{},
+			},
+			mode:     "agent",
+			expected: []string{"podmuticasts: command does not define output struct", "podmuticasts: command for controller must define one endpoint", "podmuticasts: command for flow aggregator must define one endpoint"},
+		},
+		{
+			name: "Command for flowaggregator defines non-resource endpoint",
+			cd: &commandDefinition{
+				use: "log-level",
+				flowAggregatorEndpoint: &endpoint{
+					nonResourceEndpoint: &nonResourceEndpoint{
+						params: []flagInfo{
+							{
+								name:  "",
+								usage: "Empty flag",
+								arg:   true,
+							},
+							{
+								name:      "namespace",
+								usage:     "Get log-level from specific Namespace.",
+								shorthand: "na",
+							},
+							{
+								name:      "output",
+								usage:     "used to display output",
+								shorthand: "out",
+							},
+						},
+					},
+				},
+				controllerEndpoint: &endpoint{},
+				agentEndpoint:      &endpoint{},
+			},
+			mode:     "flowaggregator",
+			expected: []string{"log-level: command does not define output struct", "log-level: command for agent must define one endpoint", "log-level: command for controller must define one endpoint", "log-level: flag name cannot be empty", "log-level: length of a flag shorthand cannot be larger than 1: na", "log-level: flag redefined: output", "log-level: length of a flag shorthand cannot be larger than 1: out"},
+		},
+		{
+			name: "Command for flowaggregator defines resource endpoint",
+			cd: &commandDefinition{
+				use: "podmuticasts",
+				flowAggregatorEndpoint: &endpoint{
+					resourceEndpoint: &resourceEndpoint{
+						namespaced: true,
+					},
+				},
+				controllerEndpoint: &endpoint{},
+				agentEndpoint:      &endpoint{},
+			},
+			mode:     "flowaggregator",
+			expected: []string{"podmuticasts: command does not define output struct", "podmuticasts: command for agent must define one endpoint", "podmuticasts: command for controller must define one endpoint"},
+		},
+	}
+	for _, tt := range tc {
+		t.Run(tt.name, func(t *testing.T) {
+			runtime.Mode = tt.mode
+			errs := tt.cd.validate()
+			strErrors := make([]string, len(errs))
+			for i, err := range errs {
+				strErrors[i] = err.Error()
+			}
+			assert.Equal(t, tt.expected, strErrors)
+		})
+	}
+}
+func TestTableOutputForQueryEndpoint(t *testing.T) {
+	policyRef0 := network_policy.PolicyRef{"testNamespace", "test-ingress-egress", "uid-1"}
```

Comment:
Ditto, check the golangci result. You should provide key as well:
```
network_policy.PolicyRef{Namespace: "testNamespace", Name: "test-ingress-egress", UID: "uid-1"}
```

Reply:
Fixed